src/Illuminate/Console/Scheduling/Event.php

```diff
@@ -687,11 +690,11 @@ public function getSummaryForDisplay()
      * @param  \DateTime|string  $currentTime
      * @param  int  $nth
      * @param  bool  $allowCurrentDate
-     * @return \Illuminate\Support\Carbon
+     * @return \Carbon\Carbon
      */
     public function nextRunDate($currentTime = 'now', $nth = 0, $allowCurrentDate = false)
     {
-        return Carbon::instance(CronExpression::factory(
+        return app(Factory::class)->instance(CronExpression::factory(
             $this->getExpression()
         )->getNextRunDate($currentTime, $nth, $allowCurrentDate, $this->timezone));
```

Comment:
Console component doesn't load `app()` which is under `Illuminate\Foundation`, nor does it require `illuminate/container`.